src/export/exporter.ts

```diff
@@ -42,16 +56,36 @@ export abstract class AzureMonitorBaseExporter implements BaseExporter {
       };
 
       if (err) {
+        // Request failed -- always retry
         this._logger.error(err.message);
         this._persister.push(envelopes, persistCb);
-      } else if (isRetriable(statusCode)) {
+      } else if (statusCode === 200) {
+        // Success -- @todo: start retry timer
+        if (!this._retryTimer) {
```

Comment:
Why you need this timer?, if 200 you can try to send persisted data every time if any

Reply:
I think the thought is that if it failed due to some networking error, some lag time is given for the (ingestion server or client server) network to fully scale back up since we would be in a state of sending 2x envelopes.